airflow/providers/microsoft/psrp/operators/psrp.py

```diff
@@ -146,6 +146,10 @@ def execute(self, context: "Context") -> Optional[List[Any]]:
         if ps.had_errors:
             raise AirflowException("Process failed")
 
+        rc = ps.runspace_pool.host.rc
+        if rc:
+            raise AirflowException("Process exited with non-zero status code: %d", rc)
```

Comment:
```suggestion
            raise AirflowException(f"Process exited with non-zero status code: {rc}")
```

This is not logging, there’s no `%`-formatting.